contributing/code_of_conduct/care_team.rst

```diff
@@ -21,7 +21,7 @@ Members
 
 Here are all the members of the CARE team (in alphabetic order). You can contact
 any of them directly using the contact details below or you can also contact all
-of them at once by emailing **care@symfony.com**:
+of them at once by emailing care@symfony.com.
```

Comment:
In order to use the bolded link, it must be written as follow:

```
of them at once by emailing |careEmail|_:
.. _careEmail: care@symfony.com
.. |careEmail| replace:: **care@symfony.com**
```

Reply:
I'll try this, thanks.

Reply:
Unfortunately, that seems to have the same effect.

Reply:
I now used `** care@symfony.com **` (with spaces around it). That seems to work okay.

Reply:
See this link:

http://rst.ninjs.org/#b2YgdGhlbSBhdCBvbmNlIGJ5IGVtYWlsaW5nIHxjYXJlRW1haWx8XzoKCi4uIF9jYXJlRW1haWw6IGNhcmVAc3ltZm9ueS5jb20KLi4gfGNhcmVFbWFpbHwgcmVwbGFjZTo6ICoqY2FyZUBzeW1mb255LmNvbSoq

Reply:
It's probably something in the builder that works differently.
After building the documentation with your suggestion, it results in the same html as previously.

Reply:
GitHub uses MarkDown and Symfony documents uses RST files, so that's why you do not see a good show on the GitHub site